test cases/d/9 features/meson.build

```diff
@@ -1,5 +1,15 @@
 project('D Features', 'd', default_options : ['debug=false'])
 
+dc = meson.get_compiler('d')
+
+# GDC 13 hard errors if options are given number values.
+# https://github.com/mesonbuild/meson/pull/11996
+if dc.get_id() == 'gdc' and dc.version().version_compare('>=13')
+    number_options_supported = true
+else
+    number_options_supported = false
+endif
```

Comment:
This conditional looks backwards, it's setting to true only when both gdc and the version is 13.

Reply:
It turns out that the ID for gdc is `gcc`, not `gdc`. Fixed.